### DISCUSSION TITLE
Kubernetes Deployment

### DISCUSSION THREAD

**Discussion by andrew-bauer-calibrated** [microsoft/aspire#1195] [Ideas]
Are there any plans for a Kubernetes deployment process?

**Comment by ReubenBond** [comment]
Hi @andrew-bauer-calibrated, here is the issue discussing progress towards Kubernetes deployment support: https://github.com/dotnet/aspire/issues/830

**Reply by davidfowl** [reply]
https://github.com/prom3theu5/aspirational-manifests

**Reply by andrew-bauer-calibrated** [reply]
Aspirate is exactly what I was looking for. Thank you!
